### ISSUE TITLE
When I set up this template according to the readme it doesn't render

### ISSUE THREAD

**Issue by RMHogervorst** [satRdays/satRday_site_template#12]
Hi @DaveParr 	When I set this website up, locally or on netlify it doesn't seem to work. Is there a setting I haven't toggled? https://github.com/satRdays/amsterdam2019

**Comment by DaveParr** [comment]
Hi @RMHogervorst , sorry you're having trouble. I'm able to clone the repo you link, and on the master branch execute `hugo serve` in my bash terminal and then go to local host and see this:

<img width="1280" alt="Screenshot 2019-08-31 at 14 35 24" src="https://user-images.githubusercontent.com/8363743/64064710-e377ff00-cbfc-11e9-9a4d-a248610797b9.png">
(I changed the first line to a hello world to check the build wasn't caching from a pervious run)

What exactly have you done and what is the specific error you are seeing? You will need hugo installed to run it: https://gohugo.io/getting-started/installing/

**Comment by RMHogervorst** [comment]
let me try again, I didn't use `hugo serve` I think, I just used `hugo`

**Comment by RMHogervorst** [comment]
What i see is a completely empty page, looking at the source I see only this

```
<pre>
</pre>

```

**Comment by RMHogervorst** [comment]
and my terminal tells me:
```
WARN 2019/09/08 12:11:02 Found no layout for "home", language "en", output format "HTML": create a template below /layouts with one of these filenames: index.en.html.html, home.en.html.html, list.en.html.html, index.html.html, home.html.html, list.html.html, index.en.html, home.en.html, list.en.html, index.html, home.html, list.html, _default/index.en.html.html, _default/home.en.html.html, _default/list.en.html.html, _default/index.html.html, _default/home.html.html, _default/list.html.html, _default/index.en.html, _default/home.en.html, _default/list.en.html, _default/index.html, _default/home.html, _default/list.html
```

**Comment by RMHogervorst** [comment]
and 

hugo serve
Building sites … WARN 2019/09/08 12:10:43 Found no layout for "taxonomyTerm", language "en", output format "HTML": create a template below /layouts with one of these filenames: categories/terms.en.html.html, categories/list.en.html.html, categories/terms.html.html, categories/list.html.html, categories/terms.en.html, categories/list.en.html, categories/terms.html, categories/list.html, taxonomy/terms.en.html.html, taxonomy/list.en.html.html, taxonomy/terms.html.html, taxonomy/list.html.html, taxonomy/terms.en.html, taxonomy/list.en.html, taxonomy/terms.html, taxonomy/list.html, _default/terms.en.html.html, _default/list.en.html.html, _default/terms.html.html, _default/list.html.html, _default/terms.en.html, _default/list.en.html, _default/terms.html, _default/list.html

**Comment by RMHogervorst** [comment]
I'm using hugo v0.54.0-B1A82C61 linux/amd64 BuildDate: 2019-02-01T09:40:34Z

**Comment by DaveParr** [comment]
So that suggests to me that your hugo install is not able to find the assets to generate the site. That's what the warn is saying: "I can't find the layouts or the home page". 

What is the location your calling the command in? It should be in the root of the project you're trying to generate, so for me it's `~ /Documents/satRdays/amsterdam2019`.

**Comment by DaveParr** [comment]
OK, actual fix: https://discourse.gohugo.io/t/found-no-layout-for-taxonomyterm-language-en-output-format-html/16376

Seems an update around December leads to this message. Still doesn't explain the lack of output on your side, but does explain the message. The fix is a quick config line, I'll add it to the core theme in a bit.

**Comment by DaveParr** [comment]
OK, I have corrected the `WARN` messages on my side that mirrored yours, which I believe were created from 'recent' (last 8 months) updates to the hugo framework. All the way through though my site was actually generating, the `WARN` messages seem to be exceptions, so I'm not really sure why yours isn't being rendered. I was working from [this commit of your repo](https://github.com/satRdays/amsterdam2019/commit/7fed03ade19316d0014a236c6362eb5d047213f0) the whole time. I was also using [this version of the theme submodule](https://github.com/satRdays/hugo-satrdays-theme/commit/6b6ec0eea4444dea29f03ce3fc2b6c06d056373f). Let's see if we can use [`hugo serve`](https://github.com/satRdays/hugo-satrdays-theme/commit/6b6ec0eea4444dea29f03ce3fc2b6c06d056373f) to sort it out. Can you please confirm the following to see if we can sort out your website generation core issue?

1. where you run the `hugo serve` command from?
2. what the full output of the `hugo server` command is?
3. what you see when you go to the locally hosted link in the output?
4. if you are using the same main repo commit and sub-module commit as me?
5. if your submodule is actually containing your work from the PR you sent for the tweet this button?

**Comment by RMHogervorst** [comment]
I did a fresh install on a new computer

Using hugo 0.58

- git clone from amsterdam2019 repo
- [1] ran hugo serve from '/Users/roelhogervorst/Documents/projecten/amsterdam2019/'
- [2]:

```
Building sites … WARN 2019/09/09 18:39:41 found no layout file for "HTML" for "taxonomyTerm": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2019/09/09 18:39:41 found no layout file for "HTML" for "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2019/09/09 18:39:41 found no layout file for "HTML" for "taxonomyTerm": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

                   | EN
+------------------+----+
  Pages            |  3
  Paginator pages  |  0
  Non-page files   |  0
  Static files     |  0
  Processed images |  0
  Aliases          |  0
  Sitemaps         |  1
  Cleaned          |  0

Total in 6 ms
Watching for changes in /Users/roelhogervorst/Documents/projecten/amsterdam2019/{content,data}
Watching for config changes in /Users/roelhogervorst/Documents/projecten/amsterdam2019/config.toml
Environment: "development"
Serving pages from memory
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop
WARN 2019/09/09 18:39:50 found no layout file for "HTML" for "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
```

- [3] Locally I see this:
```
.
├── LICENSE
├── README.md
├── config.toml
├── content
├── data
│   ├── projects
│   │   ├── Talk01.yaml
│   │   └── Talk02.yaml
│   └── speakers
│       ├── Speaker01.toml
│       └── Speaker02.toml
├── netlify.toml
├── resources
│   └── _gen
│       ├── assets
│       └── images
└── themes
    ├── hugo-agency-theme
    └── hugo-satrdays-theme

11 directories, 8 files
```
- [4] I don't actually know what version of the submodule I am using. How do I find that out? I did not change it anyway

**Comment by RMHogervorst** [comment]
[4] 
git submodule status:

```
-90e7bf048bceb82be56304b5ec7030822ffa9547 themes/hugo-agency-theme
-6b6ec0eea4444dea29f03ce3fc2b6c06d056373f themes/hugo-satrdays-theme
```

(so yes, it is the same commit in the submodule)

[5] I don't think so, given that I use the same commit

**Comment by RMHogervorst** [comment]
I found out the problem!

I didn't update the submodules

`git submodule update --init`

**Comment by DaveParr** [comment]
So is it building now and if you go to the URL in your messages 'localhost1313' do you see the website?

**Comment by RMHogervorst** [comment]
yes than the website just works

**Comment by RMHogervorst** [comment]
I think `git submodule update` is enough

**Comment by RMHogervorst** [comment]
of course the website on `https://eloquent-aryabhata-b2ad74.netlify.com/` is ugly as hell because the base url is not mismatched it's looking for resources at amsterdam2019.satrdays.org

**Comment by DaveParr** [comment]
So to close this off @RMHogervorst , could you submit a (tiny) PR update to the readme.md to outline the step that I've missed that fixed your issue? I'm guessing it's either going to be a new step 5 in administrator tasks, or a new step 1 in organiser tasks.

Once we've done that, I can close this off, and help out future me :)

**Comment by RMHogervorst** [comment]
on it!
